irislibay/malikhain
# Organized CSS and JS files

#11 

- Added named in routes/web.php
- Used those named some routes I found
- Moved particle.js to Laravel Mix so that it compiles instead of downloading via CDN'
- Moved particle.js css related files into particles.scss/css so that some pages can have particle.js while others don't (I'm assuming it's supposed to be like that)

Test:
1. `npm run dev`
2. Visit workshop. The particle.js should be there working and in fullscreen.
3. Visit `/workshop/styletransferArt` and `/workshop/styletransferPoem`, they should have particle.js too. All other pages should not have particle.js
